variables.tf

```diff
@@ -70,6 +83,12 @@ variable "node_sensor_mode" {
   }
 }
 
+variable "falcon_kpa" {
+  type        = bool
+  description = "Whether to deploy the Falcon Kubernetes Protection Agent to the cluster."
+  default     = true
+}
+
```

Comment:
The reason there are submodules in this module is to allow a CX to choose to deploy one or the other. If they just call the main module we do both.